src/promptflow/tests/sdk_cli_test/e2etests/test_flow_run.py

```diff
@@ -1117,3 +1118,30 @@ def test_get_details_against_partial_completed_run(self, pf: PFClient) -> None:
         for _, row in details2.iterrows():
             if str(row["outputs.output"]) != "(Failed)":
                 assert int(row["inputs.number"]) == int(row["outputs.output"])
+
+    # TODO: remove this patch after executor switch to default spawn
```

Comment:
Why do we need to spawn here?

Reply:
with fork process, there is a known executor issue that some line runs might timeout, while this test has strict assertions; executor has plan to switch to spawn process in the near future, so Heyi suggest this workaround to avoid flaky test.